### PR TITLE
NC-653

### PR DESCRIPTION
**Pull request by Joeper214**
ref: https://skyarch.backlog.jp/view/NC-653

### CODE REVIEW

**Comment by Joeper214** [comment]
Please review and Merge PR. @root-beer

File: app/models/zabbix.rb [hunk 1]
```diff
@@ -404,8 +404,12 @@ def get_user_id(username)
     user_info = @sky_zabbix.user.get(
       filter: { alias: username }
     )
+    unless user_info.first['userid'].nil?
```
Comment (root-beer) [inline app/models/zabbix.rb]: `user_info.first.nil?`, right?

**Comment by root-beer** [comment]
```
[16/10/14 15:57:42.952531] [INFO]:  [SkyZabbix 200 0.413891007] user.delete([nil])
[16/10/14 15:57:42.953922] [ERROR]: #<SkyZabbix::Jsonrpc::Error: Application error. No permissions to referred object or it does not exist!> from skyhopper/vendor/bundle/ruby/2.2.0/gems/sky_zabbix-2.2.0.1.2/lib/sky_zabbix/jsonrpc.rb:99:in `request'
[16/10/14 15:57:42.954268] [DEBUG]: skyhopper/vendor/bundle/ruby/2.2.0/gems/sky_zabbix-2.2.0.1.2/lib/sky_zabbix/jsonrpc.rb:99:in `request'
skyhopper/vendor/bundle/ruby/2.2.0/gems/sky_zabbix-2.2.0.1.2/lib/sky_zabbix/jsonrpc.rb:20:in `post'
skyhopper/vendor/bundle/ruby/2.2.0/gems/sky_zabbix-2.2.0.1.2/lib/sky_zabbix/client/target_base.rb:41:in `_query'
skyhopper/vendor/bundle/ruby/2.2.0/gems/sky_zabbix-2.2.0.1.2/lib/sky_zabbix/client/target_gen.rb:19:in `block (4levels) in <top (required)>'
skyhopper/app/models/zabbix.rb:433:in `delete_user'
skyhopper/app/controllers/users_admin_controller.rb:177:in `block in destroy'
```
I think we should not delete if the user does not exist.

**Comment by Joeper214** [comment]
Yes. I think so too. Is this error occurs in this [LINE](https://github.com/skyarch-networks/skyhopper/blob/b5391b42a09ebfe437ae78332a6fbd81fa0f6b7b/app/models/zabbix.rb#L433)

**Comment by root-beer** [comment]
Yes.

**Comment by Joeper214** [comment]
Please Review and Merge PR. @root-beer thanks! :bow:

**Comment by root-beer** [comment]
Do you mind if I correct the Japanese locale?

**Comment by Joeper214** [comment]
Yes please. 🙇

File: app/models/chef_server/deployment.rb [hunk 2]
```diff
@@ -108,14 +108,17 @@ def create_zabbix(stack_name, region, keypair_name, keypair_value)
...
-      ZabbixServer.create(
+      zb = ZabbixServer.create(
         fqdn: set.zabbix_fqdn,
         username: 'admin',
         password: 'ilikerandompasswords',
         version: '2.2.9',
         details: 'Default Zabbix Server for Skyhopper System'
       )
 
+      prj.project.zabbix_server_id = zb.id
```
Comment (root-beer) [inline app/models/chef_server/deployment.rb]: I think `prj.project` is not exist.

Comment (Joeper214) [inline app/models/chef_server/deployment.rb]: Yes. Thank you for the comment. I'll change that.

**Comment by root-beer** [comment]
LGTM! Thanks!

**Comment by Joeper214** [comment]
🙇

**Pull request merged**